frontend/src/pages/modelRegistrySettings/ModelRegistriesPermissions.tsx

```diff
@@ -121,6 +125,8 @@ const ModelRegistriesManagePermissions: React.FC = () => {
               }
               roleBindingPermissionsRB={{ ...roleBindings, data: filteredRoleBindings }}
               groups={groups}
+              createRoleBinding={createModelRegistryRoleBinding}
+              deleteRoleBinding={deleteModelRegistryRoleBinding}
```

Comment:
I'm actually kinda surprised typescript allows us to assign `(roleBindingName: string) => Promise<K8sStatus>` to type `(name: string, namespace: string) => Promise<K8sStatus>`, but considering the extra arg will just be ignored in the MR case this seems fine